src/mqt/bench/benchmarks/random.py

```diff
@@ -1,4 +1,4 @@
-"""Random benchmark definition."""
+"""Random benchmark definition."""  # noqa: A005
```

Comment:
I'd rather prefer to rename this file to something more suitable like `random_circuit` instead of ignoring the warning.

Reply:
I thought about that but was unsure because we would need to re-generate MQTBench since also the name of the benchmark when calling `get_benchmark` would change.
Similarly, we would need to update `mqt.bench.viewer` including the backend logic which would either lead to a major version update because of breaking changes or multiple checks at different places to avoid the breaking change and make both names work. 

Do you see any other way? To me it feels like this issue should be treated whenever we want to re-generate MQTBench (including a major version update) and for now I would be in favor of the ignore.

Reply:
I see the point. Not worth the effort here.
Let's do this if we start moving away from qiskit algorithms as a whole, which will probably be a breaking change in any case.